tests/console-backend-service/internal/resource/namespace.go

```diff
@@ -0,0 +1,51 @@
+package resource
```

Comment:
You have functions for namespace here, please remove it -> https://github.com/kyma-project/kyma/blob/master/tests/console-backend-service/internal/configurer/namespace.go